filament/src/details/IndirectLight.h

```diff
@@ -57,13 +58,51 @@ class FIndirectLight : public IndirectLight {
     static math::float3 getDirectionEstimate(const math::float3 sh[9]) noexcept;
     static math::float4 getColorEstimate(const math::float3 sh[9], math::float3 direction) noexcept;
 
+    IndirectLight::Mode getTechnique() const noexcept;
+
+    void setCenter(const math::float3& inIblCenter) noexcept;
+    const math::float3& getCenter() const noexcept;
+
+    void setHalfExtents(const math::float3& inIblHalfExtents) noexcept;
+    const math::float3& getHalfExtents() const noexcept;
+
+    void setTintAndStrength(const math::float4& inIblTintAndStrength) noexcept;
+    const math::float4& getTintAndStrength() const noexcept;
+
+    void setSphereProxy(const filament::math::float4& sphere);
+    filament::math::float4 getSphereProxy() const;
+
+    void setBoxProxy(const filament::Box& box);
+    filament::Box getBoxProxy() const;
+
+    void clearProxy();
+
 private:
     FTexture const* mReflectionsTexture = nullptr;
     FTexture const* mIrradianceTexture = nullptr;
     std::array<math::float3, 9> mIrradianceCoefs;
     float mIntensity = DEFAULT_INTENSITY;
     math::mat3f mRotation;
     uint8_t mLevelCount = 0;
+    IndirectLight::Mode iblTechnique = IndirectLight::Mode::IBL_INFINITE;
+
+    // Center of the sphere or box IBL geometry.
+    math::float3 iblCenter = math::float3(0.0f);
+
+    /**
+     * If iblTechnique is IBL_INFINITE, this property is unused.
+     *
+     * If iblTechnique is IBL_FINITE_SPHERE, then
+     *      .x is the radius of the sphere
+     *      .y is the reciprocal of the radius of the sphere
+     *
+     * If iblTechnique is IBL_FINITE_BOX, then the half extents of the box along the X, Y, Z axes.
+     */
+    math::float3 iblHalfExtents = math::float3(100.0f);
+
+    // The color (.rgb) and the intensity (.a) of the IBL tint.
+    math::float4 iblTintAndStrength = math::float4(0.0f);
```

Comment:
Please use the proper naming convention (`m` prefix, camel case).
Also why is the default value of halfExtents 100?
Use this syntax:
  `math::type mName{ value };`